modules/core/src/v2/trading/trade.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @prettier
+ */
+export interface Trade {
+  id?: string;
+  externalId: string;
+  baseAccountId: string;
+  quoteAccountId: string;
+  timestamp: Date;
+  status: TradeStatus;
+  baseAmount: string;
```

Comment:
maybe make notes that amount is in base units? I don't know if this is normal for the sdk to do.

Reply:
It's normal throughout our entire platform but never hurts to remind people :+1: